requirements/prod.in

```diff
@@ -2,7 +2,7 @@ ansi2html
 boto3
 coloredlogs
 cumulusci
-django<3.2
+django<4
```

Comment:
I spent so much time ranting about indentation that I forgot to mention that a comment explaining the pin would be helpful.

Sorry.